plugins/identity/app/models/identity/aws_credential.rb

```diff
@@ -0,0 +1,5 @@
+module Identity
+  class AWSCredential < Core::ServiceLayer::Model
```

Comment:
This should be also changed to Credential or?

Reply:
Let us keep it this way because it is specific to AWS/EC2.

Reply:
Short question… where are you actually using the model AWSCredential??
in your aws_credentials you are not mapping to the model?
https://github.com/sapcc/elektra/pull/1013/files#diff-828919a30a0401afc91327539613ecd306bc81a20a0795c36149d9fdc7875aefR14
See an example hier:
https://github.com/sapcc/elektra/blob/master/plugins/identity/app/services/service_layer/identity_services/project.rb#L16

Reply:
If no needed just remove the model.